qlib/finco/workflow.py

```diff
@@ -104,12 +104,8 @@ def run(self, prompt: str) -> Path:
             if not cfg.continous_mode:
                 res = t.interact()
             t.summarize()
-            if isinstance(t, WorkflowTask) or isinstance(t, PlanTask):
```

Comment:
Why was this removed? Will we use a stack to manage the task list or will other mechanisms be used?